Samples/PowerShell/Groups.ps1

```diff
@@ -26,6 +26,28 @@ $swis = Connect-Swis -host $hostname -cred $cred
 # Creating a new group with initial Cisco and Windows devices.
 #
 
+<# Filter samples w/ all operators:
+
+IP_Address is 192.168.10.10
+filter:/Orion.Nodes[IP_Address='192.168.10.10')]
+
+IP_Address is not 192.168.10.10
+filter:/Orion.Nodes[IP_Address!='192.168.10.10']
+
+IP_Address starts with 192.168.10.
+filter:/Orion.Nodes[StartsWith(IP_Address,'192.168.10.')]
+
+IP_Address ends with .10.%
+filter:/Orion.Nodes[EndsWith(IP_Address,'10.%')]
```

Comment:
I think this should include a period before the 10.

Reply:
Hey Dan,

Sorry about the delayed reply. I finally had 10 minutes of spare time to actually test everything. I'll update the pull request to fix that period and then it looks like it should be good.

Here's the code I used to test:

`# If you are adding multiple members:
$members = @(
    @{ Name = "IP_Address is 192.168.10.10"; Definition = "filter:/Orion.Nodes[IP_Address='192.168.10.10']" },
    @{ Name = "IP_Address is not 192.168.10.10"; Definition = "filter:/Orion.Nodes[IP_Address!='192.168.10.10']" },
    @{ Name = "IP_Address starts with 192.168.10."; Definition = "filter:/Orion.Nodes[StartsWith(IP_Address,'192.168.10.')]" },
    @{ Name = "IP_Address ends with .10.%"; Definition = "filter:/Orion.Nodes[EndsWith(IP_Address,'.10.%')]" },
    @{ Name = "IP_Address contains .10."; Definition = "filter:/Orion.Nodes[Contains(IP_Address,'.10.')]" },
    @{ Name = "IP_Address matches 192.168.*"; Definition = "filter:/Orion.Nodes[Pattern(IP_Address,'192.168.%')]" }
)

# Create group & dynamic queries
$groupId = (Invoke-SwisVerb $swis "Orion.Container" "CreateContainer" @(
    # group name
    "Sample PowerShell Group",

    # owner, must be 'Core'
    "Core",

    # refresh frequency
    60,

    # Status rollup mode:
    # 0 = Mixed status shows warning
    # 1 = Show worst status
    # 2 = Show best status
    0,

    # group description
    "Group created by the PowerShell sample script.",

    # polling enabled/disabled = true/false (in lowercase)
    "true",

    # group members
    ([xml]@(
       "<ArrayOfMemberDefinitionInfo xmlns='http://schemas.solarwinds.com/2008/Orion'>",
       [string]($members |% {
         "<MemberDefinitionInfo><Name>$($_.Name)</Name><Definition>$($_.Definition)</Definition></MemberDefinitionInfo>"
         }
       ),
       "</ArrayOfMemberDefinitionInfo>"
    )).DocumentElement
  )).InnerText`